DESCRIPTION

```diff
@@ -59,4 +59,4 @@ Encoding: UTF-8
 Language: en-US
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.1
+RoxygenNote: 7.2.2
```

Comment:
Looks like GH Actions [doesn't like this version](https://github.com/insightsengineering/teal.data/actions/runs/3740976714/jobs/6349987680).

Reply:
Dinakar was supposed to install the latest `roxygen2` but ran into some problems. It still works, though, tested in another package. And it can't be reversed, even if I downgrade my own `rozygen2`.